Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertGisTopography.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Objects.BuiltElements.Revit;
+using Objects.Utils;
+using Objects.GIS;
+using Speckle.Core.Models;
+using Speckle.Core.Models.GraphTraversal;
+
+using Objects.BuiltElements;
+
+namespace Objects.Converter.Revit
+{
+  public partial class ConverterRevit
+  {
+    public ApplicationObject GisTopographyToNative(GisTopography gisTopography)
+    {
+      var meshes = new List<Geometry.Mesh>();
+      foreach (Geometry.Mesh displayMesh in gisTopography.displayValue)
+      {
+        meshes.Add(displayMesh);
+      }
+
+      var speckleTopography = new Objects.BuiltElements.Topography()
+      {
+        applicationId = gisTopography.applicationId ??= Guid.NewGuid().ToString(),
```

Comment:
This seems odd, why do we need an invented `applicationId` here? This property can be null and is only intended to store the original id of the application this was sent from.

If it had no id, that is also ok. I think our code is prepared to use the `Base.id` if `Base.applicationId` is `null` for identification purposes.